gcc_omnibus_build_script.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/sh
+
+set -ex
+
+omnibus_git_name="omnibus-codethink-toolchain"
+omnibus_git_source="https://github.com/CodethinkLabs/$omnibus_git_name.git"
+omnibus_gcc_git_ref=""
+omnibus_source_path="/home/`whoami`/$omnibus_git_name"
```

Comment:
if you are going to use whoami better use $(whoami) although I might suggestyou check $HOME and $USER environment variables.